packages/CollabCodeEdit-Core.package/CCEMessage.class/class/clientMsgTypes.st

```diff
@@ -8,4 +8,4 @@ clientMsgTypes
 			#sendMulticastToSession:from: -> #sendMulticastToSession:from: .
```

Comment:
Please rename it to `clientMessageTypes`

Reply:
I'm a lot more interested in why we need this method at all

Reply:
I guess it has a lot of senders. If you manage to make this method obsolete, go ahead. Otherwise rename it

Reply:
I took them out